mapusaurus/api/views.py

```diff
@@ -89,7 +93,12 @@ def census(request):
 
 def tractCentroids(request):
     """This endpoint returns census tract centroids used to determine circle position on map"""
-    return HttpResponse(json.dumps(tract_centroids_as_json(request)), content_type='application/json')
+    geos = get_censustract_geos(request)
+    if geos:
+        tracts_geo_json = geo_as_json(geos)
+        return HttpResponse(json.dumps(tracts_geo_json), content_type='application/json')
```

Comment:
Wouldn't it make sense to call format_geo_to_geojson(geos) directory, avoiding an unnecessary json.loads+json.dumps?